package/generateScss.mjs

```diff
@@ -23,42 +23,71 @@ import { CustomMediaHelper } from './CustomMediaHelper.mjs';
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const customMediaHelper = new CustomMediaHelper(CustomMedia);
 
-let generatedScss = '@use "sass:list";\n';
+const openPropFiles = {
+  CustomMedia,
+  Sizes,
+  Colors,
+  ColorsHSL,
+  Shadows,
+  Aspects,
+  Borders,
+  Fonts,
+  Easings,
+  Gradients,
+  Svg,
+  Zindex,
+  MaskEdges,
+  MaskCornerCuts,
+};
 
-Object.entries({
-	...Sizes,
-	...Colors,
-	...ColorsHSL,
-	...Shadows,
-	...Borders,
-	...Fonts,
-	...Easings,
-	...Gradients,
-	...Svg,
-	...Zindex,
-	...MaskEdges,
-	...MaskCornerCuts,
-}).forEach(([key, value]) => {
-	if (key.includes('@')) {
-		return;
-	}
-	key = key.replace('--', '$');
-	generatedScss += `${key}: ${value};\n`;
-});
+const writeSCSSModule = async (moduleName, content) => {
+  const outFile = path.join(__dirname, `${moduleName}.scss`);
+  await fs.writeFile(outFile, content, { encoding: 'utf-8' });
+};
 
-Object.entries(Aspects).forEach(([key, value]) => {
-	key = key.replace('--', '$');
-	if (value.includes('/')) {
-		value = `list.slash(${value.replace('/', ',')})`; // fix sass deprecation warning: https://sass-lang.com/documentation/breaking-changes/slash-div
-	}
-	generatedScss += `${key}: ${value};\n`;
-});
+const generateSCSSModule = async (moduleName, importObj) => {
+  let generatedScss = '';
+  
+  if (moduleName.toLowerCase() === 'aspects') {
+    generatedScss = '@use "sass:list";\n';
+    
+    Object.entries(importObj).forEach(([key, value]) => {
+      key = key.replace('--', '$');
+      if (value.includes('/')) {
+        value = `list.slash(${value.replace('/', ',')})`; // fix sass deprecation warning: https://sass-lang.com/documentation/breaking-changes/slash-div
+      }
+      generatedScss += `${key}: ${value};\n`;
+    });
+    
+  } else if (moduleName.toLowerCase() === 'custommedia') {
+    Object.keys(importObj).forEach((queryName) => {
+      const processedQuery = customMediaHelper.process(queryName);
+      queryName = queryName.replace('--', '$');
+      generatedScss += `${queryName}: '${processedQuery}';\n`;
+    });
+    
+  } else {
+    Object.entries(importObj).forEach(([key, value]) => {
+      if (key.includes('@')) {
+        return;
+      }
+      key = key.replace('--', '$');
+      generatedScss += `${key}: ${value};\n`;
+    });
+  }
 
-Object.keys(CustomMedia).forEach((queryName) => {
-	const processedQuery = customMediaHelper.process(queryName);
-	queryName = queryName.replace('--', '$');
-	generatedScss += `${queryName}: '${processedQuery}';\n`;
+  await writeSCSSModule(moduleName, generatedScss);
+};
+
+Object.entries(openPropFiles).forEach(([moduleName, importObj]) => {
+  generateSCSSModule(moduleName.toLowerCase(), importObj);
```

Comment:
instead of making it all lowercase, can the names be made to match the original names from open-props? for example `media` instead of `custommedia`, `colors-hsl` instead of `colorshsl`

Reply:
I was actually thinking the same thing myself. I'll adjust it so that it includes the hyphens in the module names.

Reply:
this is better! but i can still see some inconsistencies, like `masks-corner-cuts` vs `masks.corner-cuts`

i think it would be easier to just maintain a manual mapping of file names rather than relying on the import name.